### PR TITLE
etcd-runner: move string generation to pkg/stringutil

### PR DESCRIPTION
**Pull request by xiang90**
/cc @heyitsanthony @sinsharat

### CODE REVIEW

File: pkg/stringutil/stringutil.go [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+import "math/rand"
+
+const (
+	chars = "abcdefghijklmnopqrstuvwxyz0123456789"
+)
+
+// UniqueStrings retruns a slice of randomly generated unique strings.
+func UniqueStrings(maxlen uint, n int) []string {
+	exist := make(map[string]bool)
```
Comment (heyitsanthony) [inline pkg/stringutil/stringutil.go]: `map[string]struct{}`?

**Comment by heyitsanthony** [comment]
not really convinced this needs to be a full fledged pkg but lgtm

**Comment by xiang90** [comment]
@heyitsanthony 

> not really convinced this needs to

There are other key generation funcs in other pkgs. for example:
https://github.com/coreos/etcd/blob/master/tools/benchmark/cmd/util.go#L78

i want to move them to a one pkg. this is the first step.

**Comment by sinsharat** [comment]
LGTM.

**Pull request merged**